### PR TITLE
Use preview name for helm preview Chart

### PR DESCRIPTION
**Pull request by Chemaclass**
## 📚  Description

When running `helm lint` I got an error saying that the chart name should be the same as the directory where the Chart is located. Also, a warning saying that the `apiVersion` was missing.

### CODE REVIEW

**Comment by andytson-inviqa** [comment]
What version of helm are you using? I can't recreate the chart name issue on v3.8.2, but I'm aware this used to be an issue, which is why we never integrated helm lint, instead kubeval

**Comment by andytson-inviqa** [comment]
What version of helm are you using? I can't recreate this issue on v3.8.2, but I'm aware this used to be an issue, which is why we never integrated helm lint, instead kubeval

**Comment by Chemaclass** [comment]
I am using helm v2.17.0

File: src/spryker/helm/preview/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,7 @@
 apiVersion: v1
-name: app-preview
+name: preview
 description: Base helm chart for the app-preview environment
 version: 0.0.1
+apiVersion: v1
```
Comment (andytson-inviqa) [inline src/spryker/helm/preview/Chart.yaml]: duplicated now

Comment (Chemaclass) [reply src/spryker/helm/preview/Chart.yaml]: ach, sorry! I resolved the conflict via the web browser... I will fix it from PhpStorm 😂

**Comment by andytson-inviqa** [comment]
preview charts are no longer supported directly by the harness, however it wont stop projects which need to implement this. Additionally helm v2 support has been removed

**Pull request closed without merging**